overlord/overlord.go

```diff
@@ -167,6 +165,50 @@ func (o *Overlord) Stop() error {
 	return err1
 }
 
+// Settle runs first a state engine Ensure and then wait for activities to settle.
+// That's done by waiting for all managers activities to settle while
+// making sure no immediate further Ensure is scheduled. Chiefly for tests.
+// Cannot be used in conjuction with Loop.
+func (o *Overlord) Settle() error {
+	func() {
```

Comment:
Yes, I'd personally have done without given how simple what's inside it looks, but didn't find it offensive either.